docs/source/user_guide/timeseries.ipynb

```diff
@@ -0,0 +1,374 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train_dates, test_dates = X.Date < \"1990-01-01\", X.Date >= \"1990-01-01\"\n",
+    "X_train, y_train = X.ww.loc[train_dates], y.ww.loc[train_dates]\n",
+    "X_test, y_test =  X.ww.loc[test_dates], y.ww.loc[test_dates]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Visualizing the training set"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import plotly.graph_objects as go"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = [\n",
+    "    go.Scatter(\n",
+    "        x=X_train[\"Date\"],\n",
+    "        y=y_train,\n",
+    "        mode=\"lines+markers\",\n",
+    "        name=\"Temperature (C)\",\n",
+    "        line=dict(color=\"#1f77b4\"),\n",
+    "    )\n",
+    "]\n",
+    "# Let plotly pick the best date format.\n",
+    "layout = go.Layout(\n",
+    "    title={\"text\": \"Min Daily Temperature, Melbourne 1980-1989\"},\n",
+    "    xaxis={\"title\": \"Time\"},\n",
+    "    yaxis={\"title\": \"Temperature (C)\"},\n",
+    ")\n",
+    "\n",
+    "go.Figure(data=data, layout=layout)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Running AutoMLSearch\n",
+    "\n",
+    "`AutoMLSearch` for time series problems works very similarly to the other problem types with the exception that users need to pass in a new parameter called `problem_configuration`.\n",
+    "\n",
+    "The `problem_configuration` is a dictionary specifying the following values:\n",
+    "\n",
+    "* **forecast_horizon**: The number of time periods we are trying to forecast. In this example, we're interested in predicting weather for the next 7 days, so the value is 7.\n",
+    "\n",
+    "* **gap**: The number of time periods between the end of the training set and the start of the test set. For example, in our case we are interested in predicting the weather for the next 7 days with the data as it is \"today\", so the gap is 0. However, if we had to predict the weather for next week with the data as it was on the previous Friday, the gap would be 2. It is important to select a value that matches the realistic delay between the forecast date and the most recently avaliable data that can be used to make that forecast.\n",
+    "\n",
+    "* **max_delay**: The maximum number of rows to look in the past from the current row in order to compute features. In our example, we'll say we can use the previous week's weather to predict the current week's.\n",
+    "\n",
+    "* **date_index**: The column of the training dataset that contains the date corresponding to each observation. Currently, this parameter is only used by some time-series specific models so in this example, we are passing in `None`.\n",
```

Comment:
ARIMA actually doesn't perform well on this dataset. I'm going to file an issue to investigate further.

Reply:
Added a comment to #2611 instead